pkg/3scale/amp/operator/apicast_options_provider_test.go

```diff
@@ -68,7 +68,7 @@ func testApicastProductionPodLabels() map[string]string {
 		"com.redhat.component-type":    "application",
 		"com.redhat.component-version": "nightly",
 		"com.redhat.product-name":      "3scale",
-		"com.redhat.product-version":   "master",
+		"com.redhat.product-version":   "2.11",
```

Comment:
This changes surprise me.
In PR https://github.com/3scale/3scale-operator/pull/600 a refactor was done to get the values from variables.
Has this branch been rebased from master before this PR?

Reply:
No, but I can do it to save some changes

Reply:
:+1: let's rebase 3scale-2.11-candidate and -stable one on top of master so we get all changes first